spec/features/visitor_can_create_account_and_get_to_their_dashboard_spec.rb

```diff
@@ -16,9 +16,10 @@
 
       within "nav" do
         expect(page).to have_content("Logged in as Bob")
-        #expect page to have profile information!
       end
 
+      expect(page).to have_button("Orders")
+      expect(page).to have_content("So your dryer played you! You've come to the right place.")
```

Comment:
As this is a test line regarding a welcome message we are going to leave it at this line length.